bindata/network/network-metrics/002-daemonset.yaml

```diff
@@ -0,0 +1,74 @@
+---
+kind: DaemonSet
```

Comment:
Why does this need to run on every node? It seems that it doesn't actually need access to the pods or the kernel for what it's doing. Can't there just be a single copy of it that accumulates the metrics for all nodes?

Reply:
The initial idea was to even have this implemented in the multus admission controller, together with the metrics it is already exposing. 
The discussion that brought to implementing this as an independent daemon is because we will be able to extend it to collect metrics related to dpdk / vfio which need to be collected on the same node.